src/ast/ddl.rs

```diff
@@ -527,6 +528,29 @@ impl fmt::Display for ColumnDef {
     }
 }
 
+/// Column definition for a view.
```

Comment:
Could you please add some examples to this doc comment showing what is allowed (and not)?

Reply:
Updated the description